README.rst

```diff
@@ -164,6 +164,16 @@ locked. For example::
         r'\.json$',   # unlock JSON API
     )
 
+LOCKDOWN_REMOTE_ADDR_EXCEPTIONS
+-------------------------------
+An optional list/tuple of IPv4 addresses to be matched agains incomming
```

Comment:
If I'm not mistaken that option should work fine with IPv6-addresses as well, so I suggest to reflect that in the README.

Reply:
After 6 hours of work, I didn't think that it will work with ipv6

Reply:
Shouldn't it just work? In the end whether IPv4 or IPv6 is used `REMOTE_ADDR` contains a string and that string is matched against the entries in `LOCKDOWN_REMOTE_ADDR_EXCEPTIONS`, isn't it?